connections/debug/debug_connection.py

```diff
@@ -54,6 +54,7 @@ def _run(self) -> None:
                 # full_arr.extend(self.config_mock_set_values())
                 # full_arr.extend(self.message_mock_set_values())
                 full_arr.extend(self.bulk_sensor_mock_random())
+                full_arr.extend(self.event_mock_set_values())
```

Comment:
```suggestion
                # full_arr.extend(self.event_mock_set_values())
```
Let's keep this commented out for now to reduce spam in the UI.